### PR TITLE
2.0 expose config

### PR DESCRIPTION
**Pull request by HarryDunham**
* Expose config to get params
* Add params to `get_data`: module, profiles, revision.
* Add response classes for `get_data`

### CODE REVIEW

File: ggsdk/sdk.py [hunk 1]
```diff
@@ -38,3 +40,16 @@ def save_dataset(self,
...
+                 profiles: typing.Union[str, typing.List[str]],
+                 revision: str):
+        """Call get data api.
+
+        :param module: Name of the module for the spring param path, e.g., nlp
+        :param profiles: List or string for the module's profile, e.g., test
+        :param revision: The revision for the associated param path, e.g., 2.0
+        """
+        return EnvironmentResponse(**json.loads(
+            self._core.get_data(module, profiles, revision).response.replace(
```
Comment (alexiudice) [inline ggsdk/sdk.py]: I think to match our current pattern and make future refactor/simplification easier 
 the program flow should be slightly altered here. 
 
 I'm thinking that the `GraphGridSdk#get_data` method makes a one-line call to return `self._core.get_data`, and creation of `EnvironmentResponse`/processing of `response` would be pushed back into the in the `GetDataApi#handler` method.  That way we keep the `GraphGridSdk` clean. 
 
 
 Also I realize you're trying to keep match Spring which is why the class is named `EnvironmentResponse`. What would you think about renaming the class `GetDataResponse` instead, signifying the class represents the response from the `data` config call?  Open to others thoughts on this.

**Review by alexiudice — CHANGES_REQUESTED** [review]

File: ggsdk/sdk.py (hunk 1 above)
Comment (HarryDunham) [reply ggsdk/sdk.py]: I think all of these changes make sense. Done!

File: ggcore/sdk_messages.py [hunk 2]
```diff
@@ -119,3 +119,25 @@ class SavaDatasetResponse(SdkServiceResponse):
     """Define class representing a save dataset api call response."""
     dataset_id: str = None
     save_path: str = None
+
+
+class PropertySource:
+    """Define class representing a source of name/value property pairs"""
```
Comment (alexiudice) [inline ggcore/sdk_messages.py]: nitpick: periods missing from this and the one below.

Comment (HarryDunham) [reply ggcore/sdk_messages.py]: good catch, fixed

**Review by alexiudice — APPROVED** [review]
Looks good! 

Tests are still a WIP on the `2.0-feature-sdkTests` branch, but the `GetDataApi` and associated sdk call will need tests written in the future.

**Review by bradnussbaum — APPROVED** [review]

**Pull request merged**